packages/components/src/range-control/stories/index.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * External dependencies
+ */
+import { number, text } from '@storybook/addon-knobs';
+
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import RangeControl from '../';
+
+export default { title: 'RangeControl', component: RangeControl };
+
+const RangeControlWithState = ( { ...props } ) => {
+	const [ value, setValue ] = useState( 5 );
```

Comment:
Have you considered offering the initial value to be configurable with knobs similar to `label` or `help`?

Reply:
See to my comment on the FontSizePicker, unless I am missing something, I don't think this would really change anything?

Reply:
Agreed upon further investigation 👍